### DISCUSSION TITLE
I wonder why version 2.10.0 was reverted.

### DISCUSSION THREAD

**Discussion by github-changhyeon** [TanStack/virtual#258] [Q&A] [answered]
I've read the issues below, but I don't quite understand what the problem is.

https://github.com/TanStack/react-virtual/issues/233#issuecomment-1017665192
https://github.com/TanStack/react-virtual/pull/219

If I install version 2.10.0 and use the windowRef option, will it cause any problems?
```
npm install react-virtual@2.10.0
```

**Comment by tannerlinsley** [comment, marked as answer]
It was reverted, because it was a breaking change for sem version

Tanner Linsley

**Reply by github-changhyeon** [reply]
thank you for your answer :)
